lib/age.rb

```diff
@@ -0,0 +1,15 @@
+class Age
+  def self.from_date(date_of_birth)
+    now = Time.now.utc.to_date
+
+    leap_year_correction =
```

Comment:
We can also do some more magical stuff:

```
relative = Time.at(Time.now - Time.parse(date_of_birth))
(relative.year - 1970) + relative.yday / 365.25
```
assumes you do not parse the date_of_birth as  Date in the caller.

:-)